src/test/java/org/folio/dcb/controller/DcbHubAfterTenantLocationsTest.java

```diff
@@ -0,0 +1,119 @@
+package org.folio.dcb.controller;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.absent;
+import static com.github.tomakehurst.wiremock.client.WireMock.equalTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.getRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.matchingJsonPath;
+import static com.github.tomakehurst.wiremock.client.WireMock.not;
+import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathEqualTo;
+import static com.github.tomakehurst.wiremock.client.WireMock.urlPathMatching;
+
+import java.util.List;
+import java.util.Map;
+import java.util.stream.IntStream;
+
+import org.folio.dcb.utils.DcbHubLocationsGroupingUtil;
+import org.junit.jupiter.api.Test;
+import org.springframework.http.HttpHeaders;
+import org.springframework.test.context.DynamicPropertyRegistry;
+import org.springframework.test.context.DynamicPropertySource;
+
+class DcbHubAfterTenantLocationsTest extends BaseIT {
+
+  @DynamicPropertySource
+  static void registerProperties(DynamicPropertyRegistry registry) {
```

Comment:
Usually, a method definition comes after the class variable definitions (static vars -> final vars -> class vars -> public methods -> private methods)